### PR TITLE
HDP-41: consistent env handling

### PR DESCRIPTION
**Pull request by mikaukora**

### CODE REVIEW

**Comment by mikaukora** [comment]
@Hi-Fi ready for review

File: README.md [hunk 1]
```diff
@@ -2,6 +2,14 @@
 
 The purpose of the system is to be able to receive any kind of IoT data of City of Helsinki in a performant, reliable and secure manner and serve it as an API for further usage.
 
+## Environments
+
+Review (PR branch): Delpoyed temporarily at <temporary name>.test.kuva.hel.ninja. See the nams from review's deploy step. Valid until the PR gets closed.
```
Comment (pjlampil) [inline README.md]: A typo "Delpoyed" 😺

**Review by pjlampil — APPROVED** [review]
Typo nitpick. Otherwise LGTM! 👍

File: README.md (hunk 1 above)
Comment (mikaukora) [reply README.md]: Thanks, fixed.

**Pull request merged**